doc/whats_new.rst

```diff
@@ -529,6 +529,62 @@ API changes summary
      (`#6660 <https://github.com/scikit-learn/scikit-learn/pull/6660>`_)
      by `Raghav RV`_.
 
+Contributors
```

Comment:
I'm pretty sure you're still on the list ;) I would argue that there are view contributors that are not code-contributors (Gael is still a code contributor, though infrequently ;), but I'm happy to change it.

I thought this was a bit more explicit then "People", but I'm happy to make it even more explicit.
We could also be more technical and say "Committers".

Gah how did I not see @jnothman's comment two days ago? Stupid github.